client/renderer.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Louis Moureaux <m_louis30@yahoo.com>
+ *
+ * SPDX-License-Identifier: GPLv3-or-later
+ */
+
+#pragma once
+
+#include <QObject>
+#include <QPainter>
+#include <QRect>
+#include <QRegion>
+#include <QSize>
+
+namespace freeciv {
+
+class renderer : public QObject {
```

Comment:
Now that we are converting more and more stuff to classes, we need to think a bit about the naming convention. We wanted to use CamelCase to fit with Qt's convention which we use everywhere. I'm approving this, but is the naming convention finalized. If yes, we need to document it in contributor's guidelines.

Reply:
I've been using `lower_snake` everywhere, which also matches existing client classes. I don't think the convention is documented anywhere or implemented, so it's not finalized :yum: We'll also want to add an automatic check.